ddsp/training/docker/README.md

```diff
@@ -87,3 +87,82 @@ There are two cluster configurations prepared:
 * `config_multiple_vms.yaml` - 4 VM configuration with 8 NVIDIA Tesla T4 GPUs. Training with this configuration and recommended parameters (*batch_size: 128, learning_rate:0.001, num_steps:15000, early_stop_loss_value:5.0*) takes around 5 hours and consumes around 44 [ML units](https://cloud.google.com/ai-platform/training/pricing#ml-units).
 
 Feel free to experiment and define your [cluster configurations](https://cloud.google.com/ai-platform/training/docs/using-gpus).
+
+### Enabling Hyperparameter Tuning
+
+In order to run a [hyperparameter tuning job on AI Platform](https://cloud.google.com/ai-platform/training/docs/hyperparameter-tuning-overview) you will have to add the following flag:
+
+```bash
+--hypertune=True
+```
+and use a configuration file that specifies the hyperparameter tuning configuration for your training job.
+You can use the existing one in `docker/utils/hypertune_configurations/config_hypertune.yaml` used for tuning the learning rate or you can learn more about adding hyperparameter configuration information to your configuration YAML file [here](https://cloud.google.com/ai-platform/training/docs/using-hyperparameter-tuning#job-config).
+
+### Enabling Tensorboard visualization
+
+To enable Tensorboard you have to authenticate to `gcloud` and run the following command:
+
+```bash
+gcloud auth login
+tensorboard --logdir=<path_to_save_dir> --port=8080
+```
+
+Then open your browser and go to `127.0.0.1:8080` to visualise your training.
+
+If you want to share your ML experiment results you can use [TensorBoard.dev](https://tensorboard.dev/).
+You have to install the latest version of TensorBoard to use the uploader:
+
+```bash
+pip install -U tensorboard
+tensorboard dev upload --logdir <path_to_save_dir> \
+    --name "(optional) <name>" \
+    --description "(optional) <description>"
+```
+
+Then view your experiment on TensorBoard.dev. Follow the link provided to view your experiment, or share it with others.
+
+To get a copy of the data uploaded to TensorBoard.dev run the following command:
+
+```bash
+tensorboard dev export --outdir OUTPUT_PATH
+```
+
+To delete an experiment you've uploaded to TensorBoard.dev run the following command. You can find your experiment id in the bottom left corner on your TensorBoard.dev link:
+
+```bash
+tensorboard dev delete --experiment_id EXPERIMENT_ID
+```
+
+### User Interaction Script:
+
+Another way to build the image and submit the training on AI Platform is to use user_interaction.py.
+To use it as a script, first add the path to where you have downloaded it and assign the execution permissions:
+
+```bash
```

Comment:
I think you can just run `python user_interaction.py` no? Seems like an easier option

Reply:
If you wrap this in a similar way to `ddsp_run` with `console_entry_point`, you could also add it as a pip install script so someone can just run it from the command line. It's nice to prefix them with ddsp since it's in the global namespace.

you could rename `user_interaction.py` to `ddsp_ai_platform.py` so that users could just type `ddsp_ai_platform` to kick something off.